sdk/nodejs/admissionregistration/v1/MutatingWebhookConfiguration.ts

```diff
@@ -94,15 +94,13 @@ import { getVersion } from "../../version";
               opts.version = getVersion();
           }
 
-          opts.additionalSecretOutputs = [
-              ...((opts && opts.additionalSecretOutputs) || []),
-
-          ];
-
-          opts.aliases = [
-              ...((opts && opts.aliases) || []),
-          ];
-
-          super(MutatingWebhookConfiguration.__pulumiType, name, props, opts);
+        let _opts = pulumi.mergeOptions(opts, {
+            additionalSecretOutputs: [
+            ],
+            aliases: [
+            ]
+        })
+
+          super(MutatingWebhookConfiguration.__pulumiType, name, props, _opts);
```

Comment:
Indentation looks off above.

Also, our lint rules should disallow use of `let` and require `const` here.

Reply:
Also - it's pretty ugly to emit all this merging and empty lists and spread across lines.  I get that this is generated code - but it is code users will see frequently on Go To Definition - are there reasonable ways to clean this all up a bit?

Reply:
Specifically I'm talking about things like this which I see all over the generated output now:

```
 let _opts = pulumi.mergeOptions(opts, {
            additionalSecretOutputs: [
            ],
            aliases: [
            ]
        })
```

Reply:
> Indentation looks off above.
> 
> Also, our lint rules should disallow use of `let` and require `const` here.

Fixed

> Also - it's pretty ugly to emit all this merging and empty lists and spread across lines. I get that this is generated code - but it is code users will see frequently on Go To Definition - are there reasonable ways to clean this all up a bit?

There may be a way to do so with mustache templates, but I couldn't figure out how when I added the auto-alias feature.